docs/sources/alerting/unified-alerting/high-availability/_index.md

```diff
@@ -0,0 +1,25 @@
++++
+title = " About alerting high availability"
+description = "High availability"
```

Comment:
@GrafanaWriter, should we take this opportunity to change TOML to YAML and add MenuTitle, add a proper description, and remove keywords?